src/Common/src/System/Security/Cryptography/DSACng.ImportExport.cs

```diff
@@ -321,7 +321,7 @@ public override DSAParameters ExportParameters(bool includePrivateParameters)
                             dsaParams.Seed = null;
                         }
 
-                        Debug.Assert(offset == 0, "offset == 0");
+                        Debug.Assert(offset == dsaBlob.Length, "offset == dsaBlob.Length");
```

Comment:
FWIW, I've gotten into the habit of including the actual values in the assert message, so that if the assert does fail, I have more info to go on as to why, e.g.
```C#
Debug.Assert(offset == dsaBlob.Length, $"Expected offset = dsaBlob.Length, got {offset} != {dsaBlob.Length}");
```